src/main/java/core/service/impl/ProcessingServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.service.impl;
+
+import core.model.Fruit;
+import core.service.ProcessingService;
+import core.storage.DataBase;
+import core.strategy.FruitStrategy;
+import java.util.List;
+
+public class ProcessingServiceImpl implements ProcessingService {
+    private static final String CSV_LINE_SPLITERATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public void process(FruitStrategy fruitStrategy, List<String> stringList) {
```

Comment:
these parameters should be fields in this class and be initialized through constructor